tests/test_extrinsics.py

```diff
@@ -0,0 +1,28 @@
+from subscrape.db.subscrape_db import SubscrapeDB
+import logging
+import subscrape
+
+def test():
+        
+    config = {
+        "kusama":{
+            "extrinsics":{
+                "crowdloan": ["create"]
```

Comment:
weird that the crowdloan **event** is `created` but the crowdloan **extrinsic** is `create`

Reply:
extrinsics are present tense, events are past tense